deps/chakrashim/core/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+## Contributing Code
```

Comment:
@panuhorsmalahti node currently does not use submodules for dependencies. It may be a good idea to document how dependencies are updated.

Currently for dependencies such as v8, libuv, or npm we take version updates as a single commit updating the state of the current directory, generally from a collaborator of the project we are downstream from. In between large updates we will sometimes backport changes as individual patch's.